Src/Support/Google.Apis.Core/GoogleApiException.cs

```diff
@@ -26,6 +26,7 @@ namespace Google
     public class GoogleApiException : Exception
     {
         private readonly string serviceName;
+        private readonly string message;
```

Comment:
As discussed, this can probably be a boolean just to indicate whether a message was supplied.

Reply:
And let's give it a leading underscore, which won't be inconsistent if we take other suggestions...